app/assets/javascripts/update_visit.coffee

```diff
@@ -1,38 +0,0 @@
-# Copyright © 2011-2016 MUSC Foundation for Research Development
```

Comment:
Most of the JS in this file was no longer necessary. Moved listener to service_calendar.js.coffee